logstash-core/src/main/java/org/logstash/common/io/ByteBufferPageIO.java

```diff
@@ -2,6 +2,7 @@
 
 import org.logstash.ackedqueue.Queueable;
 import org.logstash.ackedqueue.SequencedList;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
this doesn't seem to be used. why not have this be thrown for all of those `/** zilch **/` spaces?